clang/lib/Analysis/Lifetime.cpp

```diff
@@ -299,6 +309,9 @@ void runAnalysis(const FunctionDecl *Func, ASTContext &Context,
     return;
   if (Func->isInStdNamespace())
     return;
+  if (shouldSuppressLifetime(Func)) {
```

Comment:
nit: we usually avoid braces around if-bodies that only span a single line

Reply:
Changed it